docs/benchmarks-scripts/mssql/mssql-script.md

```diff
@@ -0,0 +1,17 @@
+# How to use MsSQL Script?
+
+In order to create a database with a certain number of tables, and a certain number of records in each of them, 
+you need to follow a few simple steps.
+
+1. Create a new database.
+2. In the **mssql-script.sql** file, you need to change a few points to create the required number of tables and records in them.
+   1. Change the value for the number of tables.
+   
+      ![](images/table_count_mssql.png)
+   2. You can add records of different sizes to the table - 50 bytes, 500 bytes, 10 kilobytes, and 100 kilobytes. To do this, 
+   you need to specify the number of records for each size.
+   
+      ![](images/rows_count_mssql.png)
+3. Execute the script with your changes for the new database. After the script finishes its work, you will receive the number of tables specified in the script, with names starting with **test_0** and ending with **test_(the number of tables minus 1)**.
```

Comment:
Can you also include the command to run the script against the database?

Reply:
For each instruction, I added a line how to run the script from the command line